source/JsMaterialX/README.md

```diff
@@ -74,21 +78,20 @@ cmake --build .
 
 
 ### Output
-After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in `./<build_folder>/source/JsMaterialX/`.
+After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in the global install directory of this project.
 
 ### Install
-To install the results into the install directory run
+To install the results into the test directory run
```

Comment:
can you also be specific here, like `source/JsMaterialX/test` . Otherwise sounds like a provided path

Reply:
Frankly, this whole section is bogus. Don't know why I even touched it. There will be another clean-up PR for this file later, where I want to remove this paragraph. I just postponed to not create conflicts with Aynur's PR.